pkg/goDB/conditions/tokenize.go

```diff
@@ -70,26 +69,19 @@ func SanitizeUserInput(conditional string) (string, error) {
 	}
 
 	// first, convert everything to lower case
-	r, err = regexp.Compile(".*")
-	if err != nil {
-		return sanitized, err
-	}
+	r = regexp.MustCompile(".*")
```

Comment:
Can we make this a package variable to avoid re-computation on each call (I know it's not expensive, but it's unnecessary and can be re-used)? It would also have the benefit of causing a crash right upon start instead of the first time the function is called (sure, unlikely that `".*"` will ever crash, but maybe someone introduces a typo later or whatever)...

Reply:
Yeah man. I saw that and thought the same. Was just too lazy to adjust all of it. In the spirit of smaller commits, are you cool with us doing this in separate PR?